src/pages/settings/Security/TwoFactorAuth/Steps/VerifyStep.js

```diff
@@ -22,7 +23,14 @@ import {defaultAccount, TwoFactorAuthPropTypes} from '../TwoFactorAuthPropTypes'
 
 const TROUBLESHOOTING_LINK = 'https://community.expensify.com/discussion/7736/faq-troubleshooting-two-factor-authentication-issues/p1?new=1';
 
-function VerifyStep({account = defaultAccount}) {
+const defaultProps = {
```

Comment:
Can we please use destructuring instead of `defaultProps`? This is going to be a requirement for components once we move them to TS so it's good to get used to doing this now.

Reply:
Ok, apologies this seems to be my mistake and prop destructuring for default props is only going to be used to TypeScript files.